content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,198 @@
+
+To configure the manual retry of a test when it fails, set:
+```
+An alert is triggered if your test fails for x minutes from any x of x locations`
+Retry x time before location is marked as failed
+```
+
+Once alert conditions are set, create a message for the alert and specify what services and team members to receive the alert notification email and click **Save Test**. You can also use [integrations][10], such as Slack, Pagerduty, webhooks, etc., to receive alert notifications.
+
+{{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure" responsive="true" style="width:90%;">}}
+
+### Test results
+
+An **API test** homepage will automatically populate after save. This page includes property information, historical graphs for response time and uptime, sample results, and all events and test results. Sample results include errors, resources, and traces. 
+
+To troubleshoot a failed test, scroll to the Test Results section and click on the **Test Results** tab. Click on the failed test, labeled as `Alert`, to view detailed test results. Review the failed assertions and response details such as returned status code, response time, and associated headers and body to resolve the issue.
+
+{{< img src="getting_started/synthetics/api-test-failure.png" alt="API test failure" responsive="true" style="width:90%;">}}
+
+Datadog also has an [APM integration with Synthetics][6] which allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use * for wildcards: `https://*.datadoghq.com/*`
+
+Alerts triggered with the APM integration populate under **Test Results**. Click the result link given to review test details and traces.
+
+## Set up your first private location
+
+Private locations allow you to monitor internal-facing applications or any private URLs that aren’t accessible from the public internet. They can also be used to create a new custom Synthetics location.
+
+The private location worker is shipped as a Docker container, so it can run on a Linux based OS or Windows OS if the Docker engine is available on your host and can run in Linux containers mode.
+
+By default, every second, your private location worker pulls your test configurations from Datadog’s servers using HTTPS, executes the test depending on the frequency defined in the configuration of the test, and returns the test results to Datadog’s servers.
+
+Once you created a private location, configuring a [Synthetics API test][#configure-the-request] from a private location is completely identical to the one of Datadog managed locations.
+
+To configure a private location:
+
```

Comment:
I understood that we did not want to list all of the commands, but can we still recommend the user to:
1. Set up a Vagrant Ubuntu 16.04 virtual machine: https://app.vagrantup.com/ubuntu/boxes/xenial64
2. Install Docker on that machine: https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce
?

Reply:
Actually, this is a good catch. I forgot to add this section in last minute because I was debating between putting it in the section above (since it mentions docker) or here. Will add.